config/settings/base.py

```diff
@@ -269,5 +270,14 @@
 }
 
 
-# Your stuff...
+# CHANNELS
 # ------------------------------------------------------------------------------
+ASGI_APPLICATION = 'config.asgi.application'
+CHANNEL_LAYERS = {
+    "default": {
+        "BACKEND": "channels_redis.core.RedisChannelLayer",
```

Comment:
What is this needed for?

Reply:
Good catch, this is no longer needed with the new implementation. This was for communicating between the consumer and the celery task that generated the key. Here's more info: https://channels.readthedocs.io/en/stable/topics/channel_layers.html